code/modules/mob/living/carbon/human/species.dm

```diff
@@ -774,6 +774,10 @@
 		. -= 1
 	if(H.status_flags & GOTTAGOREALLYFAST)
 		. -= 2
+	if(H.status_flags & GOTTAGOGONE)
+		. -= 3
```

Comment:
You can get this effect by putting GOTTAGOFAST and GOTTAGOREALLYFAST:

```
.-=1
.-=2
```

Someone will probably appreciate the spare bitflag slot.